recipes/uproc/bld.bat

```diff
@@ -0,0 +1,8 @@
+mkdir %PREFIX%\bin
+if errorlevel 1 exit 1
+
+move src\* %PREFIX%\bin
+if errorlevel 1 exit 1
+
+move src\include %PREFIX%\bin
+if errorlevel 1 exit 1
```

Comment:
Any chances of building the Windows one from source instead of re-packaging?

Reply:
I managed to build it under Windows using MSYS2. But I weren't able to replicate the steps in a Conda environment. There are some conda-forge packages that can probably used for this (I found at least [m2-msys2-runtime](https://anaconda.org/conda-forge/m2-msys2-runtime), [m2w64-gcc-libs-core](https://anaconda.org/conda-forge/m2w64-gcc-libs-core) and [m2w64-make](https://anaconda.org/conda-forge/m2w64-make)), but I didn't find any documentation for them and weren't able to figure things out by myself.

If you know how to build packages with MSYS (or know where I can find more information) it would be great if you could give me a hint.

Reply:
It took me some time, but I finally figured out a way to build the Windows package from source (using conda-forge packages). I changed the recipe accordingly.

Reply:
> It took me some time

Considering how difficult it is to build stuff on Windows. That was pretty quick!